tests/core.py

```diff
@@ -0,0 +1,65 @@
+import os
+import json
+import tempfile
+
+import pytest
+
+from server import app
+
+@pytest.fixture
+def client():
+    app.config['TESTING'] = True
+    client = app.test_client()
+
+    yield client
+
+def test_generate_address(client):
+    post_data = '{"command":"generate_address"}'
+    rv = client.post('/', data=post_data)
+    response = json.loads(rv.data)
+    assert len(response['addresses']) > 0
+
+    for address in response['addresses']:
+        assert len(address) is 81
+
+def test_get_tips_from_iri(client):
+    TIPS_TYPE_IRI_REQULAR_ALGORITHM = 0
+    post_data = {
+        "command": "get_tips",
+        "type": TIPS_TYPE_IRI_REQULAR_ALGORITHM,
+    }
+    rv = client.post('/', data=json.dumps(post_data))
+    response = json.loads(rv.data)
+
+    assert len(response['trunkTransaction']) is 81
+    assert len(response['branchTransaction']) is 81
+
+def test_get_tips_with_null(client):
+    TIPS_TYPE_TWO_NULL_IOTA_TNX = 1
+    post_data = {
+        "command": "get_tips",
+        "type": TIPS_TYPE_TWO_NULL_IOTA_TNX,
+    }
+    rv = client.post('/', data=json.dumps(post_data))
+    response = json.loads(rv.data)
+
+    assert len(response['trunkTransaction']) is 81
+    assert len(response['branchTransaction']) is 81
+
+def test_send_transfer(client):
+    tag = "PYTHONTEST"
```

Comment:
It should be specific to this package such as "SWARMTEST".

Reply:
Replace it with "SWARMTEST". 👍